chat-core/src/MongoDbPageStore.ts

```diff
@@ -0,0 +1,55 @@
+import { DatabaseConnection } from "./DatabaseConnection";
+import {
+  MakeMongoDbDatabaseConnectionParams,
+  makeMongoDbDatabaseConnection,
+  pageIdentity,
```

Comment:
Might be local only (not exported) from this file

Reply:
it's used in 2 files. making it it's own file